src/components/Footer/Footer.js

```diff
@@ -61,11 +60,7 @@ const Footer = ({
       return <View testID="blocker" />;
     }
     return (
-      <TouchableOpacity
-        testID="onPressBlock"
-        disabled={item?.isBlurredPost}
-        style={styles.btn}
-        onPress={onPressBlock}>
+      <TouchableOpacity testID="onPressBlock" disabled={item?.isBlurredPost} onPress={onPressBlock}>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The style prop is removed from the TouchableOpacity component. If there's no need for additional styling, this change is fine.
```diff
- style={styles.btn}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->